sbi/neural_nets/embedding_nets/transformer.py

```diff
@@ -651,6 +651,10 @@ def __init__(self, config):
             intermediate_size (int): hidden size of the feedforward layer
             head_dim (int): dimension key/query vectors
             attention_dropout (float): value for the dropout of the attention layer
+        Note:
+            This module now supports scalar time-series inputs. Inputs of shape
+            `(batch, seq_len)` or `(batch, seq_len, 1)` are automatically projected
+            to `feature_space_dim` before being passed through the transformer.
```

Comment:
I see the point of this note. However, since this is a rather new module that has not received a lot of attention I believe, we can just drop it.